src/modules/interactive.js

```diff
@@ -0,0 +1,20 @@
+const todoList = document.querySelector('.todo-list');
+
+export default function updateTodos(newTodos) {
+  const updatedTodos = [];
+  for (let i = 0; i < newTodos.length; i += 1) {
+    updatedTodos.push({ ...newTodos[i], id: i + 1 });
+  }
+
+  localStorage.setItem('todos', JSON.stringify(updatedTodos));
+
+  const description = updatedTodos.map((todo) => `
+  <li class="card todo-list-item ${todo.completed ? 'completed' : ''}" data-id="${todo.id}">
+    <input type="checkbox" ${todo.completed ? 'checked' : ''} class="checkbox"/>
+    <input type="text" value="${todo.description}" class="inputtext" id="${todo.id}"/>
+    <button type="button">🗑</button>
+  </li>
+`).join('');
+
+  todoList.innerHTML = description;
```

Comment:
You could extract this code into a function that generates the HTML markup for a single todo item based on the given todo object, and use it in a loop to generate the markup for all todo items in the list.